tests/distinct/tnil.nim

```diff
@@ -0,0 +1,41 @@
+discard """
+  file: "tnil.nim"
+  output: "0x1\n\nnil\n\nnil\n\n"
```

Comment:
I thought about that but feared that the test will not work correct on windows, because the file endings are pure LF while the echo command on windows should create CRLF ... imho.